ConfidentialClientTokenCache/Program.cs

```diff
@@ -64,9 +56,12 @@ static async Task Main(string[] args)
                 .WithTenantId(tenant)
                 .Build();
 
-            msalTokenCacheProvider.Initialize(app.UserTokenCache);
-            msalTokenCacheProvider.Initialize(app.AppTokenCache);
-
+            // Add a token cache (several serialization technologies possible)
+            app.UseTokenCaches(services =>
```

Comment:
One could also use the helper wrapper `app.UseInMemoryTokenCaches()` for an in memory token caches